packages/core/src/external/analytics/posthog.ts

```diff
@@ -16,29 +16,33 @@ export interface EventMessageV1 extends IdentifyMessageV1 {
   timestamp?: Date;
 }
 
+export type AnalyticsParams = Omit<EventMessageV1, "distinctId"> & { cxId: string };
+
 const defaultPostHogApiKey = Config.getPostHogApiKey();
 
-export function analytics(params: EventMessageV1, postApiKey?: string): PostHog | void {
+export function analytics(params: AnalyticsParams, postApiKey?: string): PostHog | void {
   const apiKey = postApiKey ?? defaultPostHogApiKey;
   if (!apiKey) return;
 
   const posthog = new PostHog(apiKey);
 
-  params.properties = {
-    ...(params.properties ? { ...params.properties } : undefined),
-    environment: Config.getEnvType(),
-    platform: "oss-api",
-    $set_once: {
-      cxId: params.distinctId,
+  const updatedParams: EventMessageV1 = {
+    ...params,
+    distinctId: "group_event",
+    groups: { company: params.cxId },
+    properties: {
+      ...(params.properties ? { ...params.properties } : undefined),
+      environment: Config.getEnvType(),
+      platform: "oss-api",
     },
```

Comment:
https://posthog.com/docs/product-analytics/group-analytics#advanced-server-side-only-capturing-group-events-without-a-user

How it should be setup

Reply:
from a quick glance, their docs say 
```posthog.capture({
    event: 'user_signed_up',
    distinctId: 'user_distinct_id',
    groups: { company: 'company_id_in_your_db' }
})```

should we use the word `company` or is it ok to use `customer` also?